tbx/static_src/sass/config/_mixins.scss

```diff
@@ -136,7 +136,7 @@
     $color: var(--color--text),
     $interaction-color: var(--color--text),
     $shadow: false,
-    $offset: 2px
+    $offset: 4px
```

Comment:
Do we definitely want to change the default offset for all links here? I thought it was only the larger ones of 8px that Ben wanted to change.

Reply:
My mistake on this one - reverted now